runtime/README.md

```diff
@@ -805,6 +806,11 @@ Caliptra models PAUSER callers to its mailbox as having 1 of 2 privilege levels:
 
 #### PAUSER privilege level active context limits
 
+PAUSER and Locality map 1:1. Consequently, only the single DPE Client associated
```

Comment:
I don't feel too strongly, but I would also put this under the "PAUSER privilege levels" heading. Since it's not exactly related to context limits.

Reply:
I don`t have a strong opinion here either, but I don`t have problem to move it under the "PAUSER privilege levels" heading.